io/io/inc/ROOT/TBufferMerger.hxx

```diff
@@ -93,6 +96,44 @@ public:
     */
    void SetMergeOptions(const TString& options);
 
+   /** Indicates that the file will not contains any TTree objects
```

Comment:
"will not contains"? I guess the input file *does* not contain any TTree objects? Who's supposed to call this? Is this an interface that will realistically be used?

Reply:
Yes. It is for files that contains only histograms for example.

Reply:
Actually I remember wrong.  This is actually a request to skip the TTrees if any.